src/pages/RecipeCalculator.vue

```diff
@@ -51,23 +51,39 @@
       v-for="(material, index) in materials.materials"
       :key="`material-${index}`"
     >
-      <div class="border-top border-bottom col-sm-2">
-        {{ material.quantity * parseInt(quantity) }}
-      </div>
-      <div class="border-top border-bottom col-sm-10">
-        {{ material.name }}
+      <div
+        class="border-top border-bottom col-sm-2"
+      >
+        <GameAsset
+          :height="72"
+          :image="Assets.MaterialImage(material.name.replace(/ /g, '_'))"
+          :title="material.name"
+        >
+          <template #tooltip>
+            <div v-html="material.name"></div>
+          </template>
+        </GameAsset>
       </div>
+      <div
+        class="border-top border-bottom col-sm-2 recipe-quantity"
+      >{{ (material.quantity * parseInt(quantity)).toLocaleString() }}</div>
+      <div
+        class="border-top border-bottom col-sm-6 padded-start"
+      >{{ material.name.padStart(material.name.length + material.indent * 12) }}</div>
```

Comment:
I would recommend going one of two ways with this:
- Apply a dynamic margin style instead of increasing the length of the string
- Create a component to render one item for this calculator that takes a `depth` or `indent` prop that also uses a dynamic margin/width